apps/redi-connect/src/components/organisms/RediHeroLanding.tsx

```diff
@@ -24,81 +24,79 @@ const RediHeroLanding = ({ type }: Props) => {
   }> = t(`loggedOutArea.homePage.hero.${type}.steps`, { returnObjects: true })
 
   return (
-    <>
-      <Section
-        className={classnames('hero-landing', {
-          [`hero-landing--${type}`]: type,
-        })}
-      >
-        <Container>
-          <Heading
-            size={1}
-            responsive={{
-              mobile: { textSize: { value: 2 } },
-              tablet: { textAlignment: { value: 'centered' } },
-            }}
-          >
-            {t(`loggedOutArea.homePage.hero.${type}.headline`)}
-          </Heading>
-          <Content
-            responsive={{
-              mobile: { textAlignment: { value: 'centered' } },
-              'tablet-only': { hide: { value: true } },
-            }}
-          >
-            <SVGImage
-              image={type}
-              className={classnames('hero-landing__illustration', {
-                [`hero-landing__illustration--${type}`]: type,
-              })}
+    <Section
+      className={classnames('hero-landing', {
+        [`hero-landing--${type}`]: type,
+      })}
+    >
+      <Container>
+        <Heading
+          size={1}
+          responsive={{
+            mobile: { textSize: { value: 2 } },
+            tablet: { textAlignment: { value: 'centered' } },
+          }}
+        >
+          {t(`loggedOutArea.homePage.hero.${type}.headline`)}
+        </Heading>
+        <Content
+          responsive={{
+            mobile: { textAlignment: { value: 'centered' } },
+            'tablet-only': { hide: { value: true } },
+          }}
+        >
+          <SVGImage
+            image={`hero${type.charAt(0).toUpperCase() + type.slice(1)}`}
```

Comment:
This line prevents redi connect from building (try running `yarn build redi-connect` and you'll see the error).

You're better off using an explicit "conversion" map here, something like:

```
const typeToSvgImageMap = {
  mentor: 'heroMentor',
  mentee: 'heroMentee',
}
```

Bonus points if you "type" (i.e. use typescript) to declare the _type of_ `typeToSvgImageMap` referencing the Props interface and the Props of `<SVGImage>`

Reply:
Solved, hope my solution works for you :)